python/pyspark/ml/classification.py

```diff
@@ -576,6 +577,115 @@ class GBTClassificationModel(TreeEnsembleModels):
     """
 
 
+@inherit_doc
+class NaiveBayes(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol):
+    """
+    Naive Bayes Classifiers.
+
+    >>> from pyspark.sql import Row
+    >>> from pyspark.mllib.linalg import Vectors
+    >>> df = sc.parallelize([
```

Comment:
It might be nice to use sqlContext.createDataFrame instead of sc.parallelize(...).toDF() since createDataFrame has the opportunity to create a local DataFrame.